vyper/utils.py

```diff
@@ -161,32 +161,36 @@ def in_bounds(cls, type_str, value):
 
 # Cannot be used for variable or member naming
 reserved_words = {
-    # types
-    'int128', 'uint256',
```

Comment:
why did all these type names get deleted?

Reply:
They didn't, I just noticed that https://github.com/vyperlang/vyper/pull/1741/files/2cc00f3ad18019d30a7222b629c526960da4612b#diff-8c0c8fd6a8ff510b61268f42b77583a1R145 existed, and therefore this was redundent. Perhaps I should rename `reserved_words` to `other_reserved_words` or something to that effect to make it clearer

Reply:
maybe the clearest way will be to add `| valid_global_keywords | base_types` to the end of the definition of `reserved_words`

Reply:
Yes in theory, but in practice there are different error messages in `is_varname_valid`, which is why it was done that way. But maybe it shouldn't be?

Reply:
> Yes in theory, but in practice there are different error messages in `is_varname_valid`, which is why it was done that way. But maybe it shouldn't be?

That's an astute point and something I looked over. But I wouldn't mind using a catch-all there. I think '{varname} is a reserved keyword' is plenty descriptive.

Reply:
Yeah, I'm thinking that too. Should improve our branch coverage :laughing: